compose/cli/main.py

```diff
@@ -84,6 +85,20 @@ def main():
         sys.exit(1)
 
 
+def log_api_error(e):
+    if 'client is newer than server' in e.explanation:
+        # we need JSON formatted errors. In the meantime...
+        client_version = e.explanation.split('client API version: ')[1].split(',')[0]
```

Comment:
No, we don't care about the server version, the client version is the one we're concerned about.